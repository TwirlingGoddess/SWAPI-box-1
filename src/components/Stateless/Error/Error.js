import React from 'react';
import '../Error/Error.css';

const Errors = () => {

  return (
    <div className='background'>
      <div className='videoBox'>
        <h2>Add Favorite Cards!</h2>         
      </div>
    </div>
  );
};

export default Errors;
