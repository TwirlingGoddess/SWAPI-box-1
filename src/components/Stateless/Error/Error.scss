.videoBox h2 {
  color: yellow;
  font-size: 10vh;
  margin: 30vh auto;
  position: absolute;
  text-shadow: 2px 2px #020101;
  width: 80%;
}