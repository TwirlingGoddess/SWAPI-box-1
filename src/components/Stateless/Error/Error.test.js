import React from 'react';
import { shallow, mount } from 'enzyme';
import Error from './Error';


describe ('', () => {
let error;
  beforeEach(() => {
    error = shallow(<Error />);
  });

  it('matches snapshot', () => {
    expect(error).toMatchSnapshot();
  });


});