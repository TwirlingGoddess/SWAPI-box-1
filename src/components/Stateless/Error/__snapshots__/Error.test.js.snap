// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Errors />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="videoBox"
>
        <h2>
                Erroring...
        </h2>
</div>,
      "className": "background",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h2>
          Erroring...
</h2>,
        "className": "videoBox",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Erroring...",
        },
        "ref": null,
        "rendered": "Erroring...",
        "type": "h2",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="videoBox"
>
          <h2>
                    Erroring...
          </h2>
</div>,
        "className": "background",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h2>
            Erroring...
</h2>,
          "className": "videoBox",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Erroring...",
          },
          "ref": null,
          "rendered": "Erroring...",
          "type": "h2",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
